### PR TITLE
ENH: Simplify and make helper scripts work on Windows

### PR DESCRIPTION
**Pull request by lassoan**
Instead of using shell scripts, which are hard to execute on Windows ported the scripts to Python.
Using Python also allowed the scripts to be greatly improved and simplified.

Also removed redundant storage of index files (in revision control, as release asset, and release note). Now csv file is only stored as release asset and md file is stored as release asset and node. No need to commit anything in revision control when files are added or removed.

Upload command uploads incoming files to all hashing algorithms. It can also creates all requested releases.

Download commands handles multiple versions of the same filename correctly (adds hash to the end of the filename if there are multiple versions).

### CODE REVIEW

**Comment by lassoan** [comment]
@jcfr Please check this PR. I would like to fix ModelToLabelmapTests (http://slicer.cdash.org/viewTest.php?onlyfailed&buildid=1798412) by updating baseline data using the new Python script.

**Comment by jcfr** [comment]
> Also removed redundant storage of index file

That makes sense.

File: README.md [hunk 1]
```diff
@@ -1,89 +1,62 @@
...
 
 Downloading files
 -----------------
 
-_The commands reported below should be evaluated in the same terminal session. See [Documentation conventions](#documentation-conventions)_
-
 1. Install [Prerequisites](#prerequisites)
 
-2. Execute download script specifying `<HASHALGO>` to copy the data form
+2. Run process_release_data.py script specifying your github token, download command, and hash algorithm. If multiple versions of the same filename is uploaded then the file will be downloaded with the checksum attached to the end of the filename.
```
Comment (jcfr) [inline README.md]: `filename is uploaded` -> `filename are uploaded`

File: process_release_data.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
+# Downloads release files or uploads new files in INCOMING folder as release assets.
```
Comment (jcfr) [inline process_release_data.py]: Consider running https://github.com/psf/black to format the python file

**Review by jcfr — CHANGES_REQUESTED** [review]
Thanks for working on this :pray: 

It definitely simplifies the process, this is great !

**Pull request merged**